cellcounter/settings.py

```diff
@@ -90,6 +89,32 @@
 LOGIN_REDIRECT_URL = '/'
 LOGOUT_URL = '/logout/'
 
+# New template system from Django 1.8
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [
+            # insert additional TEMPLATE_DIRS here
+            # Put strings here, like "/home/html/django_templates" or "C:/www/django/templates".
+            # Always use forward slashes, even on Windows.
+            # Don't forget to use absolute paths, not relative paths.
+        ],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.contrib.auth.context_processors.auth',
+                'django.template.context_processors.debug',
+                'django.template.context_processors.i18n',
+                'django.template.context_processors.media',
+                'django.template.context_processors.static',
+                'django.template.context_processors.tz',
+                'django.contrib.messages.context_processors.messages',
+            ],
+            'debug': DEBUG,
```

Comment:
My only reservation here is that there is now no granularity between TEMPLATE_DEBUG and DEBUG (though that was the behaviour of the previous code).

I think we should have a ```TEMPLATE_DEBUG``` environment variable which is sourced separately, and if it isn't provided, we default to ```DEBUG```).